core/src/Cuda/Kokkos_Cuda_Graph_Impl.hpp

```diff
@@ -159,10 +159,10 @@ struct GraphImpl<Kokkos::Cuda> {
     auto pred_ptr = GraphAccess::get_node_ptr(arg_pred_ref);
     KOKKOS_EXPECTS(bool(pred_ptr))
 
-    auto const& pred_cuda_node = pred_ptr->node_details_t::node;
+    auto& pred_cuda_node = pred_ptr->node_details_t::node;
     KOKKOS_EXPECTS(bool(pred_cuda_node))
 
-    auto const& cuda_node = arg_node_ptr->node_details_t::node;
+    auto& cuda_node = arg_node_ptr->node_details_t::node;
```

Comment:
```suggestion
    auto /*const*/& cuda_node = arg_node_ptr->node_details_t::node;
```

Reply:
I don't like this part as much in either case because it sort of implies that the `auto` is supposed to deduce `const`, but I'm okay with it for now.